<launch>
  <arg name="port_name"                default="/dev/ttyUSB0"/>
  <arg name="baud_rate"                default="460800"/>

  <!-- <param name="dynamixel_info"          value="$(find dynamixel_workbench_controllers)/config/basic.yaml"/> -->

  <node type="rs_controller" pkg="futaba_rs" name="rs_controller">
    <param name="port_name" value="$(arg port_name)" />
    <param name="baud_rate" value="$(arg baud_rate)" />
  </node>

</launch>
